react-app/src/modules/account/account.test.js

```diff
@@ -0,0 +1,25 @@
+import * as account from './account.js';
+
+afterEach(() => account.clearAccount());
+
+test('creates keypairs from username and password', async () => {
+  expect.assertions(4);
+  await account.constructor('$/()RUFJ)wuwe84349', 'dnefu384(/·$873hf7g')
+    .then(result => {
+      expect(result).toBeNull();
+      expect(account.getUsername()).toBe('$/()RUFJ)wuwe84349');
+      expect(btoa(String.fromCharCode.apply(null, account.getPublicKey())))
+        .toBe('6FUygceUHlKzGu/5ir4FNkmAtUF07uNeAAdp9+5hcy4=');
+      expect(btoa(String.fromCharCode.apply(null, account.getSecretKey())))
+        .toBe('m4jBZuDiCgcCz94VIWHmeVC9IsXOIavniA2pqeq5Gg0=');
+    });
+});
+
+test('encrypt and decrypt a message', () => {
```

Comment:
I would use an extra variable to prevent typos on the encrypted/decrypted result.
Also, imho, using variables to store the results of the api calls sometimes makes the tests easier to read.

Here's how I'd write this test:

```js
test('encrypt and decrypt a message', () => {
  expect.assertions(2);
  account._setKeyPair('6FUygceUHlKzGu/5ir4FNkmAtUF07uNeAAdp9+5hcy4=',
                      'm4jBZuDiCgcCz94VIWHmeVC9IsXOIavniA2pqeq5Gg0=');

  const plainText = 'sómething with ñ and ü\n!_¡?\'{[^]}}à';
  const encryptedMessage = account.encrypt(plainText);
  const decryptedMessage = account.decrypt(encryptedMessage);
  expect(typeof encryptedMessage).toBe('string');
  expect(decryptedMessage).toEqual(plainText);
});
```